src/System Application/App/Pdf/src/PDFDocument.Codeunit.al

```diff
@@ -39,4 +39,44 @@ codeunit 3110 "PDF Document"
     begin
         PDFDocumentImpl.ConvertToImage(ImageStream, ImageFormat, PageNumber);
     end;
+
+    /// <summary>
+    /// This procedure is used to get the invoice attachment stream from a PDF file.
+    /// </summary>
+    /// <param name="PdfStream">Input stream of the PDF file.</param>
+    /// <param name="TempBlob">Temporary blob to store the attachment.</param>
+    procedure GetDocumentAttachmentStream(PdfStream: InStream; TempBlob: Codeunit "Temp Blob"): Boolean
+    begin
+        exit(PDFDocumentImpl.GetDocumentAttachmentStream(PdfStream, TempBlob));
+    end;
+
+    /// <summary>
+    /// This procedure is used to get the properties of a PDF file.
+    /// </summary>
+    /// <param name="DocumentInStream">Input stream of the PDF file.</param>
+    /// <returns>Properties of the PDF file in JSON format.</returns>
+    procedure GetPdfProperties(DocumentInStream: InStream): JsonObject
```

Comment:
Magnus has already merged export image I believe. Could you add his changes to this file and add the method to get the number of pages from the PDF?
It would help if someone wants to export all images and also not to attempt to export a second page in a 1 page PDF document.